course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -85,33 +100,27 @@
                 },
             },
             lengthChange: false,
-            sAjaxDataProp: 'aaData',
+            sAjaxDataProp: 'data',
```

Comment:
This is listed as a legacy option (like all the type-prefixed stuff in datatables); do we still need it despite having `dataSrc` defined in the `ajax` callback? If so, maybe another comment would be helpful.